### PR TITLE
Fix Psalm issues from baseline

### PR DESCRIPTION
**Pull request by bdsl**
Just to make the code marginly easier to work with. Reduces count of Psalm errors from 537 to 467, a 13% reduction.

### CODE REVIEW

File: src/Application/Matching/Adapter.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ private function addAmountWithoutSavingFundingsToDB(CampaignFunding $funding, st
          * @psalm-suppress PossiblyInvalidArrayAccess
          * @psalm-suppress PossiblyFalseReference - we know incrBy will retrun an array in multi mode
          */
-        [$initResponse, $fundBalanceFractional] = $this->storage->multi()
+        [$_, $fundBalanceFractional] = $this->storage->multi()
```
Comment (NoelLH) [inline src/Application/Matching/Adapter.php:74]: Possibly more helpful for future editors to use underscore but with a more descriptive name still?

**Review by NoelLH — DISMISSED** [review]
Non-blocking comment inline re. unused underscore vars (which I think applies in 2 places in the same file). Otherwise all LGTM

File: src/Application/Matching/Adapter.php (hunk 1 above)
Comment (bdsl) [reply src/Application/Matching/Adapter.php:74]:
```suggestion
        [$_initResponse, $fundBalanceFractional] = $this->storage->multi()
```

File: src/Application/Matching/Adapter.php [hunk 2]
```diff
@@ -101,8 +101,9 @@ public function subtractAmountWithoutSavingToDB(CampaignFunding $funding, string
 
         /**
          * @psalm-suppress PossiblyFalseReference - in mulit mode decrBy will not return false.
+         * @psalm-suppress PossiblyInvalidArrayAccess - in this case we know exec returns array
          */
-        [$initResponse, $fundBalanceFractional] = $this->storage->multi()
+        [$_, $fundBalanceFractional] = $this->storage->multi()
```
Comment (bdsl) [inline src/Application/Matching/Adapter.php:106]:
```suggestion
        [$_initResponse, $fundBalanceFractional] = $this->storage->multi()
```

**Pull request merged**